### PR TITLE
feat: [CHAOS-85]: Chaos Service initial license plans

### PR DESCRIPTION
**Pull request by imrajdas**
Signed-off-by: Raj Babu Das <rajbabu.das@harness.io>
feat: [CHAOS-85]: Chaos Service initial license plans

[CHAOS-85]: https://harness.atlassian.net/browse/CHAOS-85?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/35162)

### CODE REVIEW

**Comment by imrajdas** [comment]
trigger codebasehash

**Comment by imrajdas** [comment]
trigger ti0

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/35162)

**Comment by imrajdas** [comment]
trigger ti0

File: 970-ng-commons/src/main/java/io/harness/ModuleType.java [hunk 1]
```diff
@@ -24,15 +24,13 @@ public enum ModuleType {
   @JsonProperty("CF") CF("Continuous Features"),
   @JsonProperty("CE") CE("Continuous Efficiency"),
   @JsonProperty("STO") STO("Security Testing Orchestration"),
+  @JsonProperty("CHAOS") CHAOS("Chaos Engineering"),
```
Comment (hextechpal) [inline 970-ng-commons/src/main/java/io/harness/ModuleType.java:27]: @imrajdas you need to add this at the end

**Review by yogesh-chauhan — APPROVED** [review]

**Comment by yogesh-chauhan** [comment]
trigger codebasehash

**Review by hextechpal — COMMENTED** [review]
> Reviewed 1 of 13 files at r2, 4 of 9 files at r3, 7 of 9 files at r4, 1 of 2 files at r6, 1 of 1 files at r7, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35162)* status: 0 of 1 approvals obtained.

**Review by hextechpal — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35162#-:-N8hLosf6QKqYO_jHq8V:bnfp4nl)* status: :shipit: complete! ✓

**Review by yogesh-chauhan — APPROVED** [review]

**Comment by yogesh-chauhan** [comment]
trigger codebasehash

**Comment by imrajdas** [comment]
trigger codebasehash

**Pull request merged**